src/modules/dashboard/components/ColonyHome/ColonyMembers/MembersSubsection.tsx

```diff
@@ -0,0 +1,231 @@
+import React, { useCallback } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import Maybe from 'graphql/tsutils/Maybe';
+
+import NavLink from '~core/NavLink';
+import Heading from '~core/Heading';
+import { Tooltip } from '~core/Popover';
+import Icon from '~core/Icon';
+import InviteLinkButton from '~dashboard/InviteLinkButton';
+
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import useAvatarDisplayCounter from '~utils/hooks/useAvatarDisplayCounter';
+import {
+  Colony,
+  useLoggedInUser,
+  BannedUser,
+  UserProfile,
+  ColonyContributor,
+  ColonyWatcher,
+} from '~data/index';
+import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
+import { useTransformer } from '~utils/hooks';
+import { getAllUserRoles } from '~modules/transformers';
+import { hasRoot, canAdminister } from '~modules/users/checks';
+
+import styles from './ColonyMembers.css';
+
+const MSG = defineMessages({
+  title: {
+    id: 'dashboard.ColonyHome.ColonyMembers.MembersSubsection.title',
+    defaultMessage: `{isContributorsSubsection, select,
+      true { Contributors }
+      other { Watchers }
+    } {hasCounter, select,
+        true { ({count})}
+        false {}
+      }`,
+  },
+  loadingData: {
+    id: 'dashboard.ColonyHome.ColonyMembers.MembersSubsection.loadingData',
+    defaultMessage: 'Loading members information...',
+  },
+  reputationFetchFailed: {
+    id: `dashboard.ColonyHome.ColonyMembers.MembersSubsection.reputationFetchFailed`,
+    defaultMessage: `Failed to fetch the colony's
+      {isContributorsSubsection, select,
+        true { contributors }
+        other { watchers }
+      }
+    `,
+  },
+  tooltipText: {
+    id: 'dashboard.ColonyHome.ColonyMembers.MembersSubsection.tooltipText',
+    defaultMessage: `{isContributorsSubsection, select,
+      true {Contributors are members of the Colony who have earned reputation.}
+      other { Watchers are members of the Colony
+         who currently don’t have any reputation. }
+    }`,
+  },
+  viewMore: {
+    id: 'dashboard.ColonyHome.ColonyMembers.MembersSubsection.viewMore',
+    defaultMessage: 'View more',
+  },
+});
+
+type BannedMember = Maybe<
```

Comment:
I think this should live either in `~data` or `~types`, but definitely not inside a single component

Reply:
It's actually not even used anywhere - probably leftover after a rebase. I added to one of the issues to remove it completely.